lab1_update.js

```diff
@@ -0,0 +1,164 @@
+/*jshint strict:false */
+'use strict';
+
+/* LAB 1: A Trip to Woodland Park Zoo
+
+ Welcome to Lab 1 :)
+
+ Be sure to read all the comments!
+
+ All of the instructions are inline with the assignment below. Look for the
+ word TODO in comments.  Each TODO will have a description of what is
+ required.
+
+ To run this file, type the following (in the terminal):
+
+    node lab1.js
+
+ and then press enter/return.
+
+ Instructions for turning this lab in are in the assignment description in
+ Canvas.
+
+ I'm happy to answer any questions on Slack.
+
+*/
+
+/* ----------------- Helper ------------------------------------------------
+ We've implemented a function that will help you sanity-check your code.
+*/
+
+function assert(expression, failureMessage) {
+  if (!expression) {
+    console.log('assertion failure: ', failureMessage);
+  }
+}
+
+/*
+ An assertion is something we expect to be true about the state of the
+ program at the point where the assertion is made.
+
+ Remember, state is all the variables that we've declared and their current
+ values.
+
+ Below are some examples for how to use the assert method. Tip: once you've run
+ this file a couple times to see how they work, comment them out.
+ That will help later when you run jscs.
+*/
+
+// assert(1 === 1);
+// assert(1 === 2, 'this is an assertion failure example. 1 === 2');
+
+/*------------------Assertions------------------------------------------------
+ TODO: 8 points
+ Invoke assert twice. (In other words, create two assertions like the ones above.)
+ Use any boolean expressions that you like, but preferably
+ zoo-themed.  Make one pass and one fail. In the failure message, describe why
+ it failed.
+*/
+
+assert('lion' == 'big cat');
+assert ('lion' === 'cheetah');
+
+The second assertion fails because although a lion (and a cheetah) are both big
+cats, a lion and a cheetah are not the same species.
+
+
+/* ----------------- Meerkats -------------------------------------------------
+ Meerkats make a sort of chirping noise (according to my 30 seconds of
+ research).  We're going to translate two sentences into meerkat speech.
+*/
+
+/*
+ TODO: 20 points
+ Your goal is to replace the words in the above sentences with 'chirp' The
+ assertions at the end of this section should pass when you're done.
+ Use **two** different kinds of loops to implement this. (10 points each.)
+ HINT: the 'split' method on String will be useful.
+*/
+var sentence1 = ['More', 'food', 'please.'];
```

Comment:
you need to start with these as a string, the way the assignment laid them out. Then you'll need to use something like split() to turn your string into an array of words.